board/safety/safety_subaru.h

```diff
@@ -106,6 +108,10 @@ static int subaru_rx_hook(CANPacket_t *to_push) {
       torque_driver_new = ((GET_BYTES(to_push, 0, 4) >> 16) & 0x7FFU);
       torque_driver_new = -1 * to_signed(torque_driver_new, 11);
       update_sample(&torque_driver, torque_driver_new);
+
+      int angle_meas_new = (GET_BYTES(to_push, 4, 2) & 0xFFFFU);
+      angle_meas_new = ROUND(to_signed(angle_meas_new, 16) * STEER_ANGLE_MEAS_FACTOR);
+      update_sample(&angle_meas, angle_meas_new);
```

Comment:
think it might be a good idea to think about adding some kind of test that ensures safety modes parse these values correctly (sending messages from python, testing the sample or global variable)

Reply:
are you sure it starts at byte 5?

Reply:
~are you sure it starts at byte 4?~ my bad, was looking at the command message

Reply:
I think vehicle speed is tested for angle based cars (https://github.com/commaai/panda/blob/933a19af848104d2a6e5b50e3aca441960fe7a80/tests/safety/common.py#L654), doesn't seem like angle is tested though? I can add that

Reply:
added measured angle test